index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Definitions by: ricardo-devis-agullo <https://github.com/ricardo-devis-agullo>
```

Comment:
as this https://github.com/fastify/fastify-bearer-auth/pull/29/files#r336102699 feedback I would remove this.

Your contribution will remain in github and `git blame` forever 💪🏻😃
If you found it in other plugins it is an oversight